group/group_info.py

```diff
@@ -21,12 +21,21 @@
 from group.models import Group
 
 from users.models import User
+from utils.user_info import validate_user
 from utils import user_info
 from utils.log_info import get_logger
 from django.http import Http404
 
 logger = get_logger()
 
+def can_edit_group(user, group):
+    user = validate_user(user)
+    return user.has_admin_auth() or (user == group.owner)
+
```

Comment:
The logic of ``can_edit_group`` and ``can_delete_group`` is the same.
Can you refactor it?